src/row.jl

```diff
@@ -253,14 +253,16 @@ ArrowTypes.JuliaType(::Val{CURVE_ARROW_NAME}) = Curve
 
 """
     const ClassRow = Legolas.@row("lighthouse.class@1",
-                                  class_index::Union{Int64,Symbol}
+                                   class_labels::Union{Missing,Vector{String}},
+                                   class_index::Union{Int64,Symbol})
 
 A type alias for [`Legolas.Row{typeof(Legolas.Schema("lighthouse.class@1"))}`](https://beacon-biosignals.github.io/Legolas.jl/stable/#Legolas.@row)
 representing a single column `class_index` that holds either an integer or the value
-`:multiclass`.
+`:multiclass`, and the associated class names
 """
 const ClassRow = Legolas.@row("lighthouse.class@1",
-                              class_index::Union{Int64,Symbol} = check_valid_class(class_index))
+                              class_index::Union{Int64,Symbol} = check_valid_class(class_index),
+                              class_labels::Union{Missing,Vector{String}}=missing)
```

Comment:
We want these to default to missing; with Legolas, as written this would force them to missing always:
```suggestion
                              class_labels::Union{Missing,Vector{String}}=coalesce(class_labels, missing))
```

Reply:
github being annoying but i am committing this!